config/defaults.cmake

```diff
@@ -88,7 +88,7 @@ set(METIS_DIR "${MFEM_DIR}/../metis-4.0" CACHE PATH "Path to the METIS library."
 
 set(LIBUNWIND_DIR "" CACHE PATH "Path to Libunwind.")
 
-# For sundials_nvecparhyp and nvecparallel remember to build with MPI_ENABLED=ON
+# For sundials_nvecmpiplusx and nvecparallel remember to build with MPI_ENABLED=ON
```

Comment:
For consistency with the fix in `config/defaults.mk`, let's fix the typo here too: the correct option is `MPI_ENABLE=ON` and not `MPI_ENABLED=ON`.

Reply:
Fixed.